### PR TITLE
#616 - Flymble case study

### PR DESCRIPTION
**Pull request by szczepanbarszczowski**
#616 - Flymble case study

### CODE REVIEW

File: src/components/CaseStudiesElements/BrowseMoreProjectsSection/BrowseMoreProjectsSection.module.scss [hunk 1]
```diff
@@ -1,6 +1,8 @@
 @import '~styles/base';
 
 .container {
+  @include container($gutter: false);
+  display: block;
   padding-bottom: 69px;
```
Comment (CodeMeNatalie) [inline src/components/CaseStudiesElements/BrowseMoreProjectsSection/BrowseMoreProjectsSection.module.scss]: hehe

File: src/components/CaseStudiesElements/BrowseMoreProjectsSection/BrowseMoreProjectsSection.module.scss [hunk 2]
```diff
@@ -9,7 +11,8 @@
   }
 
   .projectsWrapper {
-    @include flex;
+    @include flex(row, center, center);
+    width: 100%;
     padding-top: 120px;
```
Comment (CodeMeNatalie) [inline src/components/CaseStudiesElements/BrowseMoreProjectsSection/BrowseMoreProjectsSection.module.scss]: `@include pt(5);`

File: src/routes/Flymble/Flymble.module.scss [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+.secondSection,
+.thirdSection,
+.fourthSection,
+.sixthSection,
+.seventhSection,
+.eightSection,
+.ninethSection {
+  @include container;
+}
+
```
Comment (CodeMeNatalie) [inline src/routes/Flymble/Flymble.module.scss]: 🎖 
but add class `.moreProjectsWrapper`

File: src/routes/Flymble/Flymble.module.scss [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  @include pt(7);
+}
+
+.firstSection {
+  @include flex(column);
+  display: block;
+}
+
+.item {
+  @include flex(column, flex-start, flex-start);
```
Comment (CodeMeNatalie) [inline src/routes/Flymble/Flymble.module.scss]: `@indclude flex(column);`

File: src/routes/Flymble/Flymble.module.scss [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+  }
+
+  .itemSubName {
+    @include font(h6, $color-gray-dark, $font-family: $font-family-light);
+    letter-spacing: 0;
+    font-weight: 300;
+  }
+
+  .arrowButton {
+    padding: 0;
```
Comment (CodeMeNatalie) [inline src/routes/Flymble/Flymble.module.scss]: `@include p(0);`

File: src/routes/Flymble/assets/flymble-logo.svg [hunk 6]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="227px" height="264px" viewBox="0 0 227 264" version="1.1" xmlns="http://www.w3.org/2000/svg"
+     xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 50.2 (55047) - http://www.bohemiancoding.com/sketch -->
```
Comment (CodeMeNatalie) [inline src/routes/Flymble/assets/flymble-logo.svg]: Remove trashy code 🗑

File: src/routes/Flymble/Flymble.module.scss (hunk 3 above)
Comment (szczepanbarszczowski) [reply src/routes/Flymble/Flymble.module.scss]: +1

File: src/routes/Flymble/Flymble.module.scss (hunk 4 above)
Comment (szczepanbarszczowski) [reply src/routes/Flymble/Flymble.module.scss]: +1

File: src/routes/Flymble/Flymble.module.scss (hunk 5 above)
Comment (szczepanbarszczowski) [reply src/routes/Flymble/Flymble.module.scss]: +1

File: src/routes/Flymble/assets/flymble-logo.svg (hunk 6 above)
Comment (szczepanbarszczowski) [reply src/routes/Flymble/assets/flymble-logo.svg]: +1

**Pull request merged**